hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRenameRequestWithFSO.java

```diff
@@ -137,12 +138,17 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       OzoneFileStatus fromKeyFileStatus =
               OMFileRequest.getOMKeyInfoIfExists(omMetadataManager, volumeName,
                       bucketName, fromKeyName, 0);
+
       // case-1) fromKeyName should exist, otw throws exception
       if (fromKeyFileStatus == null) {
         // TODO: Add support for renaming open key
         throw new OMException("Key not found " + fromKeyName, KEY_NOT_FOUND);
       }
 
+      if (fromKeyFileStatus.getKeyInfo().isHsync()) {
+        throw new OMException("Open file cannot be renamed", RENAME_OPEN_FILE);
```

Comment:
Let's mention that it is hsync'ed:
```
throw new OMException("Open file cannot be renamed since it is hsync'ed: volumeName=" + volumeName
                       + ", bucketName=" + bucketName + ", key=" + fromKeyName, RENAME_OPEN_FILE);
```

Reply:
Thanks for the review! Updated.